recipes/xxsds-sdsl-lite/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,42 @@
+#include <iostream>
+#include <sdsl/bit_vectors.hpp>
```

Comment:
I think it's a better routine/hygiene to include the system/package headers before including local ones.
```suggestion
#include <sdsl/bit_vectors.hpp>
#include <iostream>
```